src/PowerShellEditorServices/Session/PowerShellContext.cs

```diff
@@ -174,6 +191,13 @@ public static Runspace CreateRunspace(PSHost psHost)
             }
 
             Runspace runspace = RunspaceFactory.CreateRunspace(psHost, initialSessionState);
+
+            // Windows PowerShell must be hosted in STA mode
```

Comment:
It might be obvious to some but it may still be worth a comment that the apartment state has to be set **before** the runspace is opened.